arch/xtensa/src/common/xtensa_dumpstate.c

```diff
@@ -168,7 +168,7 @@ static inline void xtensa_registerdump(void)
 
 void xtensa_dumpstate(void)
 {
-  struct tcb_s *rtcb = running_task();
+  FAR struct tcb_s *rtcb = running_task();
```

Comment:
```suggestion
  struct tcb_s *rtcb = running_task();
```
The `FAR` qualifier had been wiped out from the `xtensa` and `risc-v` archs some time ago since it is mandatory only for common code.

Reply:
I will redo the PR to address this

Reply:
But this seems to be not a valid statement as I see in `arch/risc-v/src`
1. `void up_tls_initialize(FAR struct tls_info_s *info)`
2. `void weak_function mpfs_spi0_select(FAR struct spi_dev_s *dev, uint32_t devid, bool selected)`
Probably need to clean-up this as well

Reply:
Done

Reply:
> But this seems to be not a valid statement as I see in `arch/risc-v/src`

The fact that new code introduced those qualifiers back does not invalidate my statement. They were completely removed from RISC-V and Xtensa archs in a past PR.

> Done

Thanks.